composer.json

```diff
@@ -13,12 +13,16 @@
         {"name": "Johannes Schmitt", "email": "schmittjoh@gmail.com"}
     ],
     "require": {
-        "php": "~5.6|~7.0"
+        "php": "~7.1"
     },
     "require-dev": {
+        "alcaeus/mongo-php-adapter": "^1.1",
         "phpunit/phpunit": "^5.7",
         "predis/predis":   "~1.0"
     },
+    "suggest": {
+        "alcaeus/mongo-php-adapter": "Required to use MongoDB driver"
```

Comment:
Just cross-referencing #142. Rewriting `MongoDBCache` would be a BC break, so the suggestion is good, but it's not a long-term solution. I'll create a pull request to deprecate the current class, we'll just have to think of some clever naming for the new cache class that uses `ext-mongodb`.